configure.ac

```diff
@@ -724,7 +735,8 @@ RKT_IF_HAS_FLAVOR([${RKT_STAGE1_FLAVORS}], [coreos,kvm],
         coreos/kvm flavor specific build parameters
 
         local CoreOS PXE image path:            '${RKT_LOCAL_COREOS_PXE_IMAGE_PATH}'
-        local CoreOS PXE image systemd version: '${RKT_LOCAL_COREOS_PXE_IMAGE_SYSTEMD_VER}'])])
+        local CoreOS PXE image systemd version: '${RKT_LOCAL_COREOS_PXE_IMAGE_SYSTEMD_VER}'
+        stage1 CoreOS board:                    '${RKT_STAGE1_COREOS_BOARD}'])])
```

Comment:
@krnowak 'board' is the terminology used by coreos for the build target.  Currently we have amd64-usr and arm64-usr.